legal-api/src/legal_api/reports/report.py

```diff
@@ -465,6 +465,7 @@ def _populate_business_info_to_filing(filing: Filing, business: Business):
             business_json = VersionedBusinessDetailsService.get_business_revision(filing.transaction_id, business)
```

Comment:
We get version data when we filing is completed. Might need to handle this scenario.

Do we need this change in NOA? if needed check line no 240

Reply:
It will finally be updated [here](https://github.com/bcgov/lear/blob/main/legal-api/src/legal_api/services/business_details_version.py#L609). So there's no need to change.